ide/che-core-ide-app/src/main/resources/org/eclipse/che/ide/CoreLocalizationConstant.properties

```diff
@@ -151,6 +154,9 @@ action.newFolder.title = Folder
 action.newFolder.description = Creates new folder
 action.newFile.title = File
 action.newFile.description = Creates new file
+action.newFile.add.to.index.title = Add file to Git index
+action.newFile.add.to.index.text = Would you like to add file \"{0}\" to Git index?
```

Comment:
Why do you use \" instead of " as https://github.com/codenvy/che-core/pull/470/files#diff-b36b0d04baa0df73a0a6484f4624da4dR68 ? As I know for screening symbols in properties files we should use ' symbol Source https://github.com/codenvy/che-plugins/pull/417#discussion_r45949614